### ISSUE TITLE
"Audio file shorter than expected from cue sheet"

### ISSUE THREAD

**Issue by tjimsk** [flacon/flacon#74]
The following cue sheet is giving me the warning "Audio file shorter than expected from cue sheet":

The audio file can be played with VLC and is 44:40 long.  

How to make sense of the warning message?

```
PERFORMER "Ai Otsuka"
TITLE "Love Jam"
FILE "Ai Otsuka - 2004 - Love Jam.ape" WAVE
  TRACK 01 AUDIO
    TITLE "Superman"
    PERFORMER "Ai Otsuka"
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "Happy Days"
    PERFORMER "Ai Otsuka"
    INDEX 00 03:48:27
    INDEX 01 03:50:45
  TRACK 03 AUDIO
    TITLE "Strawberry Jam"
    PERFORMER "Ai Otsuka"
    INDEX 00 07:34:63
    INDEX 01 07:35:30
  TRACK 04 AUDIO
    TITLE "I Love You"
    PERFORMER "Ai Otsuka"
    INDEX 00 11:54:36
    INDEX 01 11:58:72
  TRACK 05 AUDIO
    TITLE "Fan"
    PERFORMER "Ai Otsuka"
    INDEX 00 16:41:53
    INDEX 01 16:43:67
  TRACK 06 AUDIO
    TITLE "Crazy Shop"
    PERFORMER "Ai Otsuka"
    INDEX 00 21:02:07
    INDEX 01 21:05:37
  TRACK 07 AUDIO
    TITLE "Pompon"
    PERFORMER "Ai Otsuka"
    INDEX 00 24:42:61
    INDEX 01 24:43:22
  TRACK 08 AUDIO
    TITLE "Two Stars Remembrance Day"
    PERFORMER "Ai Otsuka"
    INDEX 00 27:05:00
    INDEX 01 27:06:10
  TRACK 09 AUDIO
    TITLE "Goldfish Fireworks"
    PERFORMER "Ai Otsuka"
    INDEX 00 31:11:47
    INDEX 01 31:15:27
  TRACK 10 AUDIO
    TITLE "Salty Grilled Tongue of Black Cow 735 Yen"
    PERFORMER "Ai Otsuka"
    INDEX 00 35:48:34
    INDEX 01 35:49:45
  TRACK 11 AUDIO
    TITLE "Friends"
    PERFORMER "Ai Otsuka"
    INDEX 00 40:08:66
    INDEX 01 40:11:12
```

**Comment by SokoloffA** [comment]
Please upload your audio file.

**Comment by tjimsk** [comment]
https://drive.google.com/open?id=1wG4FCHwNdCAUGTd_D4cyrRYDNzp6xlEs

**Comment by SokoloffA** [comment]
I used an old version of the monkey decoder program, and it contained a bug. The new beta version is available [here](https://github.com/flacon/flacon/releases/download/v5.0.0-beta1/Flacon_5.0.0-beta1.dmg).

**Comment by tjimsk** [comment]
Confirmed to be fixed from version 5.0.0 beta!
